SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1441 @@
+	$htmlOutput += $tmpHtmlOutput
+	#prepare vCenter Servers section"
+	$htmlOutput = $htmlOutput + '<H3>vCenter Server Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "VC" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdExpInDays,passwdNotifyEmail | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare SSO section"
+	$htmlOutput = $htmlOutput + '<H3>Single Sign On Password Policy</H3><p><font color=#6495ED>SSO FQDN: ' + $ppmVariables.ssoServerFqdn + '</font></p><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "SSO" } | select-object -ExpandProperty productAttributes | select-object -property * -ExcludeProperty FQDN,productType,driftAlarm,driftMessage | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare NSXMgr servers section"
+	$htmlOutput = $htmlOutput + '<H3>NSX Manager Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "NSXMgr" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdMinimumLength,apiPasswdMaxFailAttempts,apiPasswdMaxFailIntervalInSec,apiPasswdUnlockIntervalInSec,cliPasswdMaxFailAttempts,cliPasswdMaxFailIntervalInSec | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare Edge servers section"
+	$htmlOutput = $htmlOutput + '<H3>NSX Edge Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "NSXEdge" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdMinimumLength,cliPasswdMaxFailAttempts,cliPasswdMaxFailIntervalInSec | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare WSA servers section"
+	if($ppmVariables.skipWSA -eq $false){
+		$htmlOutput = $htmlOutput + '<H3>Workspace One Access Password Policy</H3><p><font color=#6495ED>WSA FQDN: ' + $ppmVariables.wsaFqdn + '</font></p><table class="myTable">'
+		$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "WSA" } | select-object -ExpandProperty productAttributes | select-object -property * -ExcludeProperty FQDN,productType,driftAlarm,driftMessage | ConvertTo-Html -Fragment -As Table
+		$htmlOutput += $tmpHtmlOutput
+	}
+	else{
+		$htmlOutput = $htmlOutput + '<H3>Workspace One Access Password Policy</h3><p><br><font color=>Workspace One Access Skipped, No environmental details provided</p>'
```

Comment:
"Workspace ONE Access"

Reply:
addressed in 99029d5ec8015e0c5c645e9abc7c3384e1b3091e